pkg/api/customization/logging/list_handler.go

```diff
@@ -153,7 +153,7 @@ func (h *ClusterLoggingHandler) setEmbeddedEndpoint(cl *v3.ClusterLogging) error
 	} else {
 		kibanapod := kibanapods[0]
 		if kibanapod.Status.Phase == running {
-			cl.Spec.EmbeddedConfig.KibanaEndpoint = fmt.Sprintf("%s:%v", kibanapod.Status.HostIP, esPort)
+			cl.Spec.EmbeddedConfig.KibanaEndpoint = fmt.Sprintf("http://%s:%v", kibanapod.Status.HostIP, kibanaservice.Spec.Ports[0].NodePort)
```

Comment:
any chance ports len can be == 0?

Reply:
if the length is 0, it return error in the line above(line 147)